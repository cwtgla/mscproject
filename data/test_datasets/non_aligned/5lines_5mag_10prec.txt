31.1023
5.0449
0.0138
-19.1001
-31.1023