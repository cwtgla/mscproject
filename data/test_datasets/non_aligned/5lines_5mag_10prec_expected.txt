127
255
21
193
0
138
207
233
255
255